it/src/main/java/com/google/cloud/teleport/it/pubsub/PubsubUtils.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.it.pubsub;
+
+import com.google.pubsub.v1.Subscription;
+import com.google.pubsub.v1.SubscriptionName;
+import com.google.pubsub.v1.Topic;
+import com.google.pubsub.v1.TopicName;
+
+/** Utilities to make working with Pubsub easier. */
+public final class PubsubUtils {
```

Comment:
Utilities should be package private.

Reply:
`PubsubUtils.createTestId` is being used on `PubsubToTextIT` to pass in as a testId. Let me know if you prefer it organized in a different way.

Reply:
It should be that the test has its own id (generally the test method or class name), and then the resource manager will call `createTestId`. This just makes it easier to write tests.

Reply:
Agreed, and changed the class to be package private.